src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -27,6 +30,9 @@
     private final Model model;
     private final Storage storage;
     private final ModuleListParser moduleListParser;
+    private final ContactListParser contactListParser;
```

Comment:
Perhaps for the implementation for ParserManager, we could abstract away the lower level details, ie the kinds of Parsers that exist (ModuleListParser, ContactListParser and TodoListParser). These Parsers would only be made known in ParserManager and LogicManager should not have to be aware of them.

Reply:
See above point.